### PR TITLE
New MVP boss 

### PR DESCRIPTION
**Pull request by sctnightcore**
http://ratemyserver.net/index.php?mvp=1&page=mob_db&mob_search=Search&page_num=1

wait add Boss Re-Stats

### CODE REVIEW

File: control/mon_control.txt [hunk 1]
```diff
@@ -156,37 +156,158 @@ Yellow Plant 0 0 0
 1967 -1 0 0
 
 ##### MVPs and Dangerous Monsters #####
+##MOB NAME : MVP
 Amon Ra 0 1 0
+Amdarias 0 1 0
```
Comment (lututui) [inline control/mon_control.txt]: Amdarais not Amdarias

Comment (sctnightcore) [reply control/mon_control.txt]: RMS is Amdarias hahhaah  
ttp://ratemyserver.net/index.php?mvp=1&page=mob_db&mob_search=Search&page_num=1

Comment (sctnightcore) [reply control/mon_control.txt]: Amdarias (Re-Stats)   (MG_AMDARAIS)   Mob-ID#2476
Amdarais   (MG_AMDARAIS)   Mob-ID#2476
= = I'll add them both.

**Comment by sctnightcore** [comment]
@fadreus  I think the teleport is better.

**Comment by fadreus** [comment]
@sctnightcore 
Teleport is fine, just suggesting the attack 0 > -1 to avoid bot engage in battle with boss if being attacked first. Instead of engaging with boss while try to teleport its much better to ignore boss completely & walk away while try to teleport.

**Comment by c4c1n6kr3m1** [comment]
i use -1 1 0 too

**Comment by sctnightcore** [comment]
ok @fadreus  @c4c1n6kr3m1

**Comment by alisonrag** [comment]
why cant you keep only the id?
with the id the config will be cross-server

**Comment by sctnightcore** [comment]
I keep name and ID
@alisonrag

**Comment by lututui** [comment]
Why do some mobs have multiple spaces between the name and -1 1 0?

i.e.
```Dark Illusion   -1 1 0
Detale   -1 1 0
Thanatos Maero   -1 1 0
Thanatos Odium   -1 1 0
Anubis   -1 1 0
Armaia   -1 1 0
Armeyer Dinze   -1 1 0
Isilla   -1 1 0
Rawrel   -1 1 0
Seyren   -1 1 0
Seyren Windsor   -1 1 0
Shecil   -1 1 0
Skogul   -1 1 0
Thorn of Magic   -1 1 0```

**Comment by sctnightcore** [comment]
Mvpname(tab)-1 1 0 

```
@fadreus
@sctnightcore
Teleport is fine, just suggesting the attack 0 > -1 to avoid bot engage in battle with boss if being attacked first. Instead of engaging with boss while try to teleport its much better to ignore boss completely & walk away while try to teleport.
--
```

**Comment by lututui** [comment]
Maybe we should move `control/mon_control/avoid_mvps.txt` to `tables/avoid_mvps.txt`?

That way, we wouldn't have a duped file for control-br, what do you guys think? @alisonrag @fadreus @c4c1n6kr3m1 @sctnightcore ?

**Comment by fadreus** [comment]
I personally prefer boss remain in `mon_control.txt`.
How about `xconf.pl` associated with `avoid_mvps.txt`?

In some situations I change boss setting during in-game with macro.

**Comment by fadreus** [comment]
I personally prefer boss remain in `mon_control.txt`.
Newbies aren't really encouraged to touch anything besides control since they like to mess up and said it's not working. If changed to tables, they might start touching here and there.
Wiki need to be updated too (and some new feature already not mention in wiki)

In some situations I change boss setting during in-game with macro.
How about `xconf.pl` associated with `avoid_mvps.txt`?

**Comment by sctnightcore** [comment]
I think ok  .if move to **tables/avoid_mvps.txt**

**Comment by Nipodemos** [comment]
by the way, mon_control support `!include`  ?
i always thought that only config.txt support that

@fadreus xconf.pl will work normally since !include will do his job and include all mvp inside the `%mon_control` hash, which is what the xconf changes and check

any custom change should be done in the original mon_control.txt file

**Comment by fadreus** [comment]
Sounds good.
But then again I'm wondering why make `avoid_mvps.txt` in the first place? It's purpose?

**Comment by Nipodemos** [comment]
That's a good question...and I also wonder why keep the name of mvp if there is the ID that is universal

File: tables/bRO/control-br/mon_control/avoid_mvps.txt [hunk 2]
```diff
@@ -0,0 +1,240 @@
...
+1583 -1 1 0 #Tao Gunka
+1708 -1 1 0 #Thanatos
+1312 -1 1 0 #Turtle General
+1751 -1 1 0 #Valkyrie Randgris
+1685 -1 1 0 #Vesper
+1648 -1 1 0 #Whitesmith Harword
+1917 -1 1 0 #Wounded Morroc
+1658 -1 1 0 #Ygnizem
+
+#ID mob
```
Comment (Nipodemos) [inline tables/bRO/control-br/mon_control/avoid_mvps.txt]: it is repeated here
all mobs from this line and below are repeated

**Comment by sctnightcore** [comment]
i back to fix if have free time

**Pull request closed without merging**